src/components/add-products/add-products.js

```diff
@@ -1,20 +1,36 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
+import { useSelector } from 'react-redux';
 
+import { productActions } from '../../store/products/action';
 import BuildForm from '../form-builder';
 import ProductCard from './product-card';
-import { productActions } from '../../store/products/action';
 
 import './product.scss';
 
 function AddProducts() {
+  const path = window.location.pathname;
```

Comment:
you can potentially get the path if you are using `react-router-dom` through `useLocation`
https://stackoverflow.com/questions/42253277/react-router-v4-how-to-get-current-route